src/metatensor/models/experimental/pet/tests/test_functionality.py

```diff
@@ -25,7 +27,7 @@ def test_prediction_subset():
                 unit="eV",
             )
         },
-        supported_devices=["cpu"],
+        supported_devices=["cpu", "cuda"],
```

Comment:
These should be given in order of preference
```suggestion
        supported_devices=["cuda", "cpu"],
```